### PR TITLE
Add some utility functions common for MS & PFS

### PR DESCRIPTION
**Pull request by pcppcp**

### CODE REVIEW

File: raiden_libs/no_ssl_patch.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import warnings
+import requests
+import contextlib
+
+try:
+    from functools import partialmethod
+except ImportError:
+    # Python 2 fallback: https://gist.github.com/carymrobbins/8940382
```
Comment (palango) [inline raiden_libs/no_ssl_patch.py]: py2 fallback is unnecessary

File: raiden_libs/utils/signing.py [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+
+
+def private_key_to_address(private_key: str) -> str:
+    """ Converts a private key to an Ethereum address. """
+    return to_checksum_address(
+        public_key_to_address(PrivateKey.from_hex(remove_0x_prefix(private_key)).public_key)
+    )
+
+
+def address_from_signature(sig: bytes, msg: bytes):
```
Comment (palango) [inline raiden_libs/utils/signing.py]: docstring please

**Comment by LefterisJP** [comment]
Tests fail. I just noticed we can merge anyway. Will change this.

File: raiden_libs/gevent_error_handler.py [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+IGNORE_ERROR = Hub.SYSTEM_ERROR + Hub.NOT_ERROR
+
+
+def register_error_handler(error_handler):
+
+    Hub._origin_handle_error = Hub.handle_error
+
+    def custom_handle_error(self, context, type, value, tb):
+        if not issubclass(type, IGNORE_ERROR):
+            # print 'Got error from greenlet:', context, type, value, tb
```
Comment (LefterisJP) [inline raiden_libs/gevent_error_handler.py]: Don't commit commented out code.

**Pull request merged**